api/helpers/game-states/moves/seven-points/validate.js

```diff
@@ -45,8 +45,8 @@ module.exports = {
         throw new Error('game.snackbar.seven.pickAndPlay');
       }
 
-      if (playedCard.rank > 10) {
-        throw new Error('game.snackbar.points.numberOnlyForPoints');
+      if (playedCard.rank >= 11) {
+        throw new Error('game.snackbar.sevenScuttle.mustPlaySeven');
```

Comment:
The error message should stay the same as it was. This is a pre-existing error message that's defined in all the translation files eg src/translations/en.json. 


It is an error message that applies to the points move and sevenPoints move. Both of them should use the previous version of this error string